sdk/go/common/workspace/plugins_test.go

```diff
@@ -198,58 +201,12 @@ func TestPluginSelection_ExactMatchNotFound(t *testing.T) {
 		},
 	}
 
-	requested := semver.MustParseRange("0.2.0")
-	result := SelectCompatiblePlugin(candidatePlugins, apitype.ResourcePlugin, "myplugin", requested)
+	version := semver.MustParse("0.2.0")
+	result := SelectCompatiblePlugin(candidatePlugins,
+		PluginSpec{Kind: apitype.ResourcePlugin, Name: "myplugin", Version: &version})
 	assert.Nil(t, result)
 }
 
-func TestPluginSelection_PatchVersionSlide(t *testing.T) {
```

Comment:
We don't accept version ranges anywhere, so we don't need this test anymore (deleted because now we can't pass in version ranges anymore).